unlock-app/src/components/content/MembersContent.tsx

```diff
@@ -219,7 +230,7 @@ const MetadataTableWrapper = ({
           </select>
         </span>
         <Input
-          type="text"
+          type={currentFilter?.type ?? 'text'}
```

Comment:
In case of `keyId` type is `number`, so this prevents invalid entries

Reply:
Ok good!

Reply:
I would actually keep it as text and parse it to number. Due to how react-controlled input works, it's better that way to avoid a situation where you end up with 0 in value.